recipe/git_patch.patch

```diff
@@ -0,0 +1,59 @@
+diff --git a/bin/igorbinarywave.py b/bin/igorbinarywave.py
+index 70c1bc3..0615bbf 100755
+--- a/bin/igorbinarywave.py
++++ b/bin/igorbinarywave.py
+@@ -38,5 +38,6 @@ class WaveScript (Script):
+             pprint.pprint(wave)
+ 
+ 
+-s = WaveScript(description=__doc__)
+-s.run()
++def main():
++    s = WaveScript(description=__doc__)
++    s.run()
```

Comment:
One minor suggestion: add
```
if __name__ == '__main__':
    main()
```
to the bottom, so directly running the file still does something.

Reply:
sure, that's very reasonable, thanks